### ISSUE TITLE
Add OpenLibra

### ISSUE THREAD

**Issue by cnngimenez** [libreprojects/libreprojects#176]
> The idea behind this project is to try and bring together in one place, most of the quality bibliographic resources, with a special interest in those written in the Spanish language. In this way, all that you need to do to consult a book, or easily download it to print or read on your favorite device, is come to this service and perform your searches in an easy and categorized way. [[From the About page.]](https://openlibra.com/en/page/about)

Homepage: https://openlibra.com

**Comment by airon90** [comment]
I don't think it meets the requirements:

* [ ] open-source downloadable free software
* [x] web-based & hosted and with a gratis plan
* [x] usable easy to use for everyone

**Comment by jancborchardt** [comment]
Yeah … weird that it is not open source itself, or open database. The repositories are empty: https://github.com/openlibra?tab=repositories
